src/cli/self_update/windows.rs

```diff
@@ -285,6 +282,75 @@ pub fn do_remove_from_path() -> Result<()> {
     _apply_new_path(new_path)
 }
 
+pub fn do_add_to_programs() -> Result<()> {
+    use std::path::PathBuf;
+
+    let key = RegKey::predef(HKEY_CURRENT_USER)
+        .create_subkey(r"Software\Microsoft\Windows\CurrentVersion\Uninstall\rustup")
```

Comment:
This should be `Rustup`, not `rustup`, as the project name is capitalised.

Reply:
You're talking about the registry path, right ? What about the text displayed in the list of installed programs (`rustup - Rust toolchain manager` or `Rustup - Rust toolchain manager`)

Reply:
By the way, I can't see any place where the project name is capitalised.

rustup.rs:

> rustup is an installer for...

>rustup is an official Rust project.

README.md:

>rustup: the Rust toolchain installer

Reply:
`Rustup` please. We discussed this on discord prior to my saying this in the review; I realise the prior art in the repo may be inconsistent.